### DISCUSSION TITLE
How to export correlation coefficient of each daily CCF

### DISCUSSION THREAD

**Discussion by Amituofuo** [ROBelgium/MSNoise#283] [Q&A]
Hi, Thomas Lecocq. 
With your help, I have calculated dv/v. Thank you very much. 
I noticed in your article that 'The correlation coefficient of each daily CCF with a given REF can be evaluated and is already a good indicator that something happened under, around or to ones’ sensors' 
After reading the help document, I still can't find the method to export the correlation coefficient. How can I export the correlation coefficient as a table? 
Thank you.

**Comment by asyates** [comment]
Unless I'm mistaken, there currently isn't a method within MSNoise to export the correlation coefficient. 

Instead, it's necessary to write some code that will read the daily CCFs from within the STACKS directory, and compare this with the reference CCF (also within the STACKS directory). You can compute the correlation coefficient using various python packages (e.g. scipy or numpy).
